types/otp-generator/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for otp-generator 4.0
+// Project: https://github.com/Maheshkumar-Kakade/otp-generator
+// Definitions by: Md Sifatul Islam Rabbi <https://github.com/sifatulrabbi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface Options {
+    digits?: boolean;
+    lowerCaseAlphabets?: boolean;
+    upperCaseAlphabets?: boolean;
+    specialChars?: boolean;
+}
+
+declare const _default: {
+    generate: (length: number, options: Options) => string;
```

Comment:
you need to make those params fully optional, as documented in public usage:
https://github.com/Maheshkumar-Kakade/otp-generator#generatelength-options
(and seen in the source code)